uaclient/entitlements/tests/test_base.py

```diff
@@ -1354,31 +1354,31 @@ class TestHandleRequiredPackages:
             ),
             (
                 [{"name": "package"}],
-                [mock.call("/etc/apt/sources.list")],
+                [mock.call()],
                 [mock.call(["package"])],
                 True,
             ),
             (
                 [{"name": "package"}, {"name": "package2"}],
-                [mock.call("/etc/apt/sources.list")],
+                [mock.call()],
                 [mock.call(["package", "package2"])],
                 True,
             ),
         ],
     )
-    @mock.patch("uaclient.apt.update_sources_list")
     @mock.patch("uaclient.apt.run_apt_install_command")
-    @mock.patch("uaclient.apt.run_apt_update_command")
+    @mock.patch(
```

Comment:
This change brings no loss, as now the file selection is done in a separate function, and it's tested.
Thanks for fixing this.